src/app/features/fee-editor/stacks/stacks-fees-loader.tsx

```diff
@@ -0,0 +1,63 @@
+import { useMemo } from 'react';
+
+import type { StacksTransactionWire } from '@stacks/transactions';
+
+import { useCalculateStacksTxFees } from '@leather.io/query';
+import { convertAmountToBaseUnit, createMoneyFromDecimal } from '@leather.io/utils';
+
+import { type Fee, type Fees, feePriorityTimeMap } from '../fee-editor.context';
+
+interface StacksFeesLoaderProps {
+  children(
+    fees: Fees | undefined,
+    isLoading: boolean,
+    getCustomFee: (rate: number) => Fee
+  ): React.ReactNode;
+  unsignedTx?: StacksTransactionWire;
+}
+export function StacksFeesLoader({ children, unsignedTx }: StacksFeesLoaderProps) {
+  const { data: stxFees, isLoading } = useCalculateStacksTxFees(unsignedTx);
+
+  const fees = useMemo<Fees | undefined>(() => {
+    if (!stxFees) return;
+
```

Comment:
Is this component really a "loader". The return is in the `useMemo` so we're always calling `children` even when there are no stx fees loaded, which defeats the point no?